docker/docker-compose.yaml

```diff
@@ -71,20 +71,32 @@ services:
       - CI
       - BINTRAY_USER
       - BINTRAY_KEY
+      - ORG_GRADLE_PROJECT_signingKey
+      - ORG_GRADLE_PROJECT_signingPassword
+      - SONATYPE_USER
+      - SONATYPE_TOKEN
     command: >
       bash -cl "./gradlew --no-daemon --parallel --max-workers=4 clean check &&
       ./gradlew --no-daemon bintrayUpload"
 
+# Don't run `publish` task on `publish-snapshot` for now ^^ replace with below command later.
+#      bash -cl "./gradlew --no-daemon --parallel --max-workers=4 clean check &&
+#      ./gradlew --no-daemon publish bintrayUpload"
+
   publish-release:
     << : *common
     depends_on: [runtime-setup]
     environment:
       - CI
       - BINTRAY_USER
       - BINTRAY_KEY
+      - ORG_GRADLE_PROJECT_signingKey
+      - ORG_GRADLE_PROJECT_signingPassword
+      - SONATYPE_USER
+      - SONATYPE_TOKEN
     command: >
       bash -cl "./gradlew --no-daemon --parallel --max-workers=4 -PreleaseBuild=true clean check &&
-      ./gradlew --no-daemon -PreleaseBuild=true bintrayUpload"
+      ./gradlew --no-daemon -PreleaseBuild=true publish bintrayUpload"
```

Comment:
Let's remove `bintrayUpload` from both release and snapshot publications, and remove all bintray-related dependencies and configuration. We don't need it anymore.

Reply:
Hmm not sure about removing it as part of this PR, let's just deactivate for now.

Reply:
I'm ok with removing the bintray plugin as a follow up. Let's remove it from docker files, including bintray env variables.

Reply:
I've kept the env vars for now. Keeping most of it intact will simplify rollback if we encounter issues in the near term.